src/composables/useProposalVotes.ts

```diff
@@ -112,6 +112,22 @@ export function useProposalVotes(
     return addressResolved;
   }
 
+  async function resolveLens(handle: string) {
+    console.log(':resolveEns', handle);
+    const res = await fetch('https://api.lens.dev/', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        query: `query Profiles { profiles(request: { handles: ["${handle}"], limit: 1}) { items { ownedBy }}}`
+      })
+    });
+    const result = await res.json();
+    const addressResolved = result?.data?.profiles?.items[0]?.ownedBy;
+    return addressResolved;
+  }
```

Comment:
```suggestion
```suggestion
 async function resolveLens(handle: string) {
  console.log('resolveLens:', handle);
  
  try {
    const response = await fetch('https://api.lens.dev/', {
      method: 'POST',
      headers: {
        'Content-Type': 'application/json'
      },
      body: JSON.stringify({
        query: `
          query Profiles {
            profiles(request: { handles: ["${handle}"], limit: 1 }) {
              items {
                ownedBy
              }
            }
          }
        `
      })
    });

    if (!response.ok) {
      throw new Error(`HTTP error ${response.status}`);
    }

    const result = await response.json();
    return result.data?.profiles?.items?.[0]?.ownedBy || null
  } catch (error) {
    console.error('Error in resolveLens:', error);
    return null;
  }
}

```
```